libcsl_godot/src/ledger/transaction.rs

```diff
@@ -254,39 +401,11 @@ impl Utxo {
                             .expect("UTxO Lovelace exceeds maximum")
```

Comment:
This is the last `expect` in the module and I wonder if it's worth removing it.

I think it could be somewhat easy to crash a game by just injecting a UTxO somewhere with a BigInt holding a value larger than what `u64` can hold (I think this function is currently being used for parsing UTxOs).

Maybe we could truncate the value to the maximum possible and issue a warning during conversion? It is very unlikely a UTxO of this size exists (or will exist).

Reply:
I noticed we are throwing an error in `lib.rs` when this same situation occurs. Maybe we should do the same for consistency?